README.md

```diff
@@ -119,4 +119,81 @@ module "transfer" {
   bucket_name                = aws_s3_bucket.transfer-bucket.id
   bucket_arn                 = aws_s3_bucket.transfer-bucket.arn
 }
+```
+
+## OpenVPN
+
+```
+module "vpc" {
+  source = "terraform-aws-modules/vpc/aws"
+
+  name = "myclient-${var.env}"
+  cidr = var.vpc_cidr
+
+  azs              = var.availability_zones
```

Comment:
as this is an example, we could perhaps use example values? Same for the ones below